kernel/kernel-api/src/main/java/io/delta/kernel/engine/coordinatedcommits/Commit.java

```diff
@@ -34,12 +34,12 @@ public class Commit {
 
   private final FileStatus fileStatus;
 
-  private final long commitTimestamp;
+  private final long timestamp;
```

Comment:
Any reason to change it? We also have `file_modification_timestamp` somewhere

Reply:
my intuition is: `Commit.getTimestamp` obviously refers to the timestamp of the commit. just like how `Commit.getVersion` obviously refers to the version of the commit. i.e. we don't call it `commitVersion` and `commitFileStatus`

Reply:
That's very intuitive!

Hopefully we can make other code-paths in the `delta-spark` also follow the same convention for this to be clear all around.

For e.g. https://github.com/delta-io/delta/blob/6ae4b62845ed579bb5a19f4646831c4ee2931c02/spark/src/main/scala/org/apache/spark/sql/delta/actions/actions.scala#L1165 CommitInfo has a `getCommitTimestamp` and  `getTimestamp` and the two things can be different.